### PR TITLE
update to use 2 spaces for indentation

### PR DESCRIPTION
**Pull request by nilslp**
Hotgraphic converted to use 2 spaces for indentation.

### CODE REVIEW

File: templates/hotgraphic.hbs [hunk 1]
```diff
@@ -1,26 +1,26 @@
...
-        {{#if _graphic.attribution}}
-          <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
-        {{/if}}
-        {{#each _items}}
-          <button class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}}" data-index="{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" aria-label="{{title}}.{{#if _isVisited}} {{../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
-            <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
-          </button>
-        {{/each}}
-      {{/if}}
+  <div class="hotgraphic-graphic"{{#unless _useGraphicsAsPins}} role="list"{{/unless}}>
```
Comment (guywillis) [inline templates/hotgraphic.hbs]: L4 through to L24 require indenting by one level

File: templates/hotgraphic.hbs [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
+    {{#if _useGraphicsAsPins}}
+    <div class="hotgraphic-narrative" role="list">
+      {{#each _items}}
+      <button role="listitem" class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}} hotgraphic-graphic-pin-{{@index}} {{#if _graphic._classes}}{{_graphic._classes}}{{/if}}" data-index="{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" aria-label="{{title}}.{{#if _isVisited}} {{../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+      <div class="hotgraphic-graphic-pin-image component-item-color item-{{@index}}" style="background-image: url({{_graphic.src}})"></div>
```
Comment (guywillis) [inline templates/hotgraphic.hbs]: Single indentation required on L9

File: templates/hotgraphic.hbs [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+      {{/each}}
     </div>
+    {{else}}
+    <img src="{{_graphic.src}}" aria-label="{{_graphic.alt}}" />
+    {{#if _graphic.attribution}}
+      <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
+    {{/if}}
+    {{#each _items}}
+      <button class="base hotgraphic-graphic-pin component-item-text-color {{#if _isVisited}}visited{{/if}}" data-index="{{@index}}" style="top:{{{_top}}}%; left:{{{_left}}}%;" aria-label="{{title}}.{{#if _isVisited}} {{../_globals._accessibility._ariaLabels.visited}}.{{/if}}">
+      <div class="hotgraphic-graphic-pin-icon component-item-color icon icon-pin item-{{@index}}"></div>
```
Comment (guywillis) [inline templates/hotgraphic.hbs]: Single indentation required on L20

File: templates/hotgraphicPopup.hbs [hunk 4]
```diff
@@ -1,42 +1,42 @@
...
-            </div>
-            {{#if _graphic.attribution}}
-            <div class="graphic-attribution">{{{_graphic.attribution}}}</div>
-            {{/if}}
-        </div>
+  {{#each _items}}
+  <div class="hotgraphic-item component-item clearfix {{#if _isVisited}}visited{{/if}} {{#if _isActive}}active{{/if}} {{_classes}}" data-index={{@index}}>
+    <div class="hotgraphic-item-content">
+      <div class="hotgraphic-item-content-inner">
+      <div class="hotgraphic-content-title" {{a11y_attrs_heading 'componentItem'}}>
```
Comment (guywillis) [inline templates/hotgraphicPopup.hbs]: L6 through to L11 require one indentation

File: templates/hotgraphicPopup.hbs [hunk 5]
```diff
@@ -1,42 +1,42 @@
...
+      </div>
+      <div class="hotgraphic-content-body">
+        {{{compile body}}}
+      </div>
+      </div>
     </div>
-    {{/each}}
+    <div class="hotgraphic-item-graphic">
+      <div class="hotgraphic-item-graphic-inner">
+      <img src="{{_graphic.src}}" aria-label="{{_graphic.alt}}"/>
```
Comment (guywillis) [inline templates/hotgraphicPopup.hbs]: Single indentation required on L16

**Comment by nilslp** [comment]
@guywillis can you take another look please.

**Pull request merged**